integration_tests/Makefile

```diff
@@ -1,12 +1,8 @@
-.PHONY: test-setup test
-
-test-setup: build-amid
-	docker pull p0bailey/docker-flask
-	docker pull wazopbx/wait
```

Comment:
Why removing the docker pull?
On dev environment:
* If another version of wazopbx/wait is builded to fix something and I try to run tests, I will need to search why tests don't pass because I don't have the latest image. It will add another *itch* to the integration tests

Reply:
docker-compose up will repick the new version according to the do so it should have no impact.

Reply:
`docker-compose up` do not pick the last version. If you already have the image, it will reuse it!